docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -5,14 +5,34 @@ You can establish communication between a media controller server and a media co
 
 The main features of the Media Controller API include:
 
-- Updating and retrieving information
+- Retrieving application list
 
-  You can [update the metadata and playback information](#updating-and-retrieving-information) on the server side, and then retrieve the metadata and playback information on the client side.
+  You can [retrieve the server and the client application list](#retrieving-application-list) to communicate with
+the applications you want.
+  Additionally, the client can get the latest server information even though the application is not activated currently. The last server that changed the state to "MC_PLAYBACK_STATE_PLAYING" becomes the latest server.
+
+  > **Note**
+  >
+  > This feature supports Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable.
+
+- Updating and retrieving playback
+
+  You can [update the playback information](#updating-and-retrieving-playback) on the server side, and then retrieve the playback information on the client side.
 
   The media controller server provides current information about the registered application that you can send to the client.
 
   When the client requests the information, the media controller server updates the state information of an active application before transferring the data. If the application is not running when the client request arrives, the media controller server transfers the latest information.
 
+- Updating and retrieving metadata
+
+  You can [update the metadata](#updating-and-retrieving-metadata) on the server side, and then retrieve the metadata on the client side.
+  
+  The media controller server can provide metadata of current playing content.
+  
+  > **Note**
+  >
+  > Since Tizen 5.5 metadata for Season, Episode, and Resolution are supported.
```

Comment:
@haejeongkim 
Kindly add "," after 5.5 

"Since Tizen 5.5, metadata for Season, Episode, and Resolution are supported."
Both are correct but this adds a pause and looks better

Reply:
Done